schematic/store/synapse.py

```diff
@@ -1543,11 +1543,15 @@ def _add_id_columns_to_manifest(
             else:
                 manifest["Id"] = ""
 
+        # Retrieve the ID column name (id, Id and ID) are treated the same.
+        id_col_name = [col for col in manifest.columns if col.lower() == "id"][0]
+
```

Comment:
Will the manifest **always** have a column named id(Id, or ID) at this point?

Reply:
Yes in the steps above its checking if an `id.lower()` column is in the manifest. If not it looks for a `Uuid` column and renames that to `Id`. If there is no `Uuid` then it creates a `Id` column. This is checking to find the actual case of `id`. Previously it was looking for `Id` but in some cases the formatting is different...